spec/text_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'text'
+
+describe Text do
+
+  let(:messages) { double :messages }
+  let(:account) { double :account, messages: messages }
+  let(:api) { double :api, account: account }
+  let(:client) { double :client, api: api }
+  let(:client_class) { double :client_class, new: client }
+
+  describe 'send'
+    it 'calls create on messages of client' do
+      text = described_class.new(client_class)
+      allow(messages).to receive(:create)
+      expect(messages).to receive(:create).with(an_instance_of(Hash))
```

Comment:
I think we covered this bit during the group code review.  This test could be more specific; it could specify the args that we expect `#create` to be called with.